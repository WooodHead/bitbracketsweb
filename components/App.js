import React, { Component } from 'react';
import Home from './home/Home';

class App extends Component {
  render() {
    return (
      <div>
        <div />
        <style jsx>{``}</style>
      </div>
    );
  }
}

export default App;
