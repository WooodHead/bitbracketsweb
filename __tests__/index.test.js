/* eslint-env jest */

import { shallow } from "enzyme";
import React from "react";
import renderer from "react-test-renderer";

import Router from 'next/router';
import App from '../pages/index.js';

const mockedRouter = { push: () => {}, prefetch: () => {} };
Router.router = mockedRouter;

describe('With Enzyme', () => {
  xit('App shows "Welcome Next"', () => {
    const app = shallow(<App />);

    expect(app.find('h1').text()).toEqual('Welcome to Next!');
  });
});

describe('With Snapshot Testing', () => {
  xit('App shows "Hello world!"', () => {
    const component = renderer.create(<App />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
