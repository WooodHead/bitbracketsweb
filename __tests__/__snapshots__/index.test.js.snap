// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing App shows "Hello world!" 1`] = `
<div>
  <div
    className="jsx-803102528"
  >
    <div
      className="jsx-803102528 navbarResponsive"
    >
      <nav
        className="navbar navbar-default"
        style={
          Object {
            "background": "#323150",
            "borderColor": "#323150",
            "borderRadius": 0,
            "margin": 0,
          }
        }
      >
        <div
          className="container"
        >
          <div
            className="navbar-header"
          >
            <a
              className="jsx-661576907 navbar-brand"
              href="#home"
              style={
                Object {
                  "color": "#3ffeca",
                }
              }
            >
              BitBrackets
            </a>
            <div
              className="jsx-661576907"
              style={
                Object {
                  "float": "right",
                }
              }
            >
              <div>
                <span
                  className="colorButton"
                >
                  <button
                    className="MuiButtonBase-root-16 MuiButton-root-1"
                    disabled={false}
                    onBlur={[Function]}
                    onClick={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    onKeyUp={[Function]}
                    onMouseDown={[Function]}
                    onMouseLeave={[Function]}
                    onMouseUp={[Function]}
                    onTouchEnd={[Function]}
                    onTouchMove={[Function]}
                    onTouchStart={[Function]}
                    style={
                      Object {
                        "color": "#fff",
                      }
                    }
                    tabIndex="0"
                    type="button"
                  >
                    <span
                      className="MuiButton-label-2"
                    >
                      <svg
                        aria-hidden="true"
                        className="MuiSvgIcon-root-19"
                        color={undefined}
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
                        />
                      </svg>
                    </span>
                    <span
                      className="MuiTouchRipple-root-25"
                    />
                  </button>
                </span>
              </div>
            </div>
            <div
              className="jsx-661576907"
              style={
                Object {
                  "display": "none",
                  "visibility": "hidden",
                }
              }
            >
              <button
                className="navbar-toggle collapsed"
                onClick={[Function]}
                type="button"
              >
                <span
                  className="sr-only"
                >
                  Toggle navigation
                </span>
                <span
                  className="icon-bar"
                />
                <span
                  className="icon-bar"
                />
                <span
                  className="icon-bar"
                />
              </button>
            </div>
          </div>
          <div
            aria-expanded={null}
            className="navbar-collapse collapse"
          >
            <ul
              className="nav navbar-nav"
              role={null}
              style={undefined}
            >
              <li
                className=""
                role="presentation"
                style={undefined}
              >
                <a
                  href="#"
                  onClick={[Function]}
                  onKeyDown={[Function]}
                  role="button"
                >
                  <span
                    className="jsx-661576907 "
                  />
                </a>
              </li>
              <li
                className=""
                role="presentation"
                style={undefined}
              >
                <a
                  href="#"
                  onClick={[Function]}
                  onKeyDown={[Function]}
                  role="button"
                >
                  <span
                    className="jsx-661576907 "
                  />
                </a>
              </li>
              <li
                className=""
                role="presentation"
                style={undefined}
              >
                <a
                  href="#"
                  onClick={[Function]}
                  onKeyDown={[Function]}
                  role="button"
                >
                  <span
                    className="jsx-661576907 "
                  />
                </a>
              </li>
              <li
                className=""
                role="presentation"
                style={undefined}
              >
                <a
                  href="#"
                  onClick={[Function]}
                  onKeyDown={[Function]}
                  role="button"
                >
                  <span
                    className="jsx-661576907 "
                  />
                </a>
              </li>
            </ul>
          </div>
        </div>
      </nav>
    </div>
    <div
      className="jsx-803102528 navbar"
    >
      <nav
        className="navbar navbar-default"
        style={
          Object {
            "background": "#323150",
            "borderColor": "#323150",
            "borderRadius": 0,
            "margin": 0,
          }
        }
      >
        <div
          className="container"
        >
          <div
            className="navbar-header"
          >
            <a
              className="jsx-661576907 navbar-brand"
              href="#home"
              style={
                Object {
                  "color": "#3ffeca",
                }
              }
            >
              BitBrackets
            </a>
            <div
              className="jsx-661576907"
              style={
                Object {
                  "float": "right",
                }
              }
            />
            <div
              className="jsx-661576907"
              style={
                Object {
                  "display": "none",
                  "visibility": "hidden",
                }
              }
            >
              <button
                className="navbar-toggle collapsed"
                onClick={[Function]}
                type="button"
              >
                <span
                  className="sr-only"
                >
                  Toggle navigation
                </span>
                <span
                  className="icon-bar"
                />
                <span
                  className="icon-bar"
                />
                <span
                  className="icon-bar"
                />
              </button>
            </div>
          </div>
          <div
            aria-expanded={null}
            className="navbar-collapse collapse"
          >
            <ul
              className="nav navbar-nav"
              role={null}
              style={
                Object {
                  "float": "right",
                }
              }
            >
              <li
                className=""
                role="presentation"
                style={undefined}
              >
                <a
                  href="#"
                  onClick={[Function]}
                  onKeyDown={[Function]}
                  role="button"
                >
                  <span
                    className="jsx-661576907 description"
                  >
                    Sign Up
                  </span>
                </a>
              </li>
              <li
                className=""
                role="presentation"
                style={undefined}
              >
                <a
                  href="#"
                  onClick={[Function]}
                  onKeyDown={[Function]}
                  role="button"
                >
                  <span
                    className="jsx-661576907 description"
                  >
                    Log In 
                  </span>
                </a>
              </li>
              <li
                className=""
                role="presentation"
                style={undefined}
              >
                <a
                  href="#"
                  onClick={[Function]}
                  onKeyDown={[Function]}
                  role="button"
                >
                  <span
                    className="jsx-661576907 description"
                  />
                </a>
              </li>
              <li
                className=""
                role="presentation"
                style={undefined}
              >
                <a
                  href="#"
                  onClick={[Function]}
                  onKeyDown={[Function]}
                  role="button"
                >
                  <span
                    className="jsx-661576907 description"
                  />
                </a>
              </li>
              <li
                className="dropdown"
                style={undefined}
              >
                <a
                  activekey="true"
                  aria-expanded={false}
                  aria-haspopup={true}
                  className="dropdown-toggle"
                  href="#"
                  id="basic-nav-dropdown"
                  onClick={[Function]}
                  onKeyDown={[Function]}
                  role="button"
                >
                  Language
                   
                  <span
                    className="caret"
                  />
                </a>
                <ul
                  aria-labelledby="basic-nav-dropdown"
                  className="dropdown-menu"
                  role="menu"
                >
                  <li
                    className=""
                    role="presentation"
                    style={undefined}
                  >
                    <a
                      href="#"
                      onClick={[Function]}
                      onKeyDown={[Function]}
                      role="menuitem"
                      tabIndex="-1"
                    >
                      English
                    </a>
                  </li>
                  <li
                    className=""
                    role="presentation"
                    style={undefined}
                  >
                    <a
                      href="#"
                      onClick={[Function]}
                      onKeyDown={[Function]}
                      role="menuitem"
                      tabIndex="-1"
                    >
                      Spanish
                    </a>
                  </li>
                  <li
                    className=""
                    role="presentation"
                    style={undefined}
                  >
                    <a
                      href="#"
                      onClick={[Function]}
                      onKeyDown={[Function]}
                      role="menuitem"
                      tabIndex="-1"
                    >
                      Portuguese
                    </a>
                  </li>
                  <li
                    className=""
                    role="presentation"
                    style={undefined}
                  >
                    <a
                      href="#"
                      onClick={[Function]}
                      onKeyDown={[Function]}
                      role="menuitem"
                      tabIndex="-1"
                    >
                      Italian
                    </a>
                  </li>
                </ul>
              </li>
            </ul>
          </div>
        </div>
      </nav>
      <nav
        className="navbar navbar-default"
        style={
          Object {
            "background": "#282840",
            "borderColor": "#282840",
            "borderRadius": 0,
            "margin": 0,
          }
        }
      >
        <div
          className="container"
        >
          <div
            className="navbar-header"
          >
            <a
              className="jsx-661576907 navbar-brand"
              href="#home"
              style={undefined}
            />
            <div
              className="jsx-661576907"
              style={
                Object {
                  "float": "right",
                }
              }
            />
            <div
              className="jsx-661576907"
              style={
                Object {
                  "display": "none",
                  "visibility": "hidden",
                }
              }
            >
              <button
                className="navbar-toggle collapsed"
                onClick={[Function]}
                type="button"
              >
                <span
                  className="sr-only"
                >
                  Toggle navigation
                </span>
                <span
                  className="icon-bar"
                />
                <span
                  className="icon-bar"
                />
                <span
                  className="icon-bar"
                />
              </button>
            </div>
          </div>
          <div
            aria-expanded={null}
            className="navbar-collapse collapse"
          >
            <ul
              className="nav navbar-nav"
              role={null}
              style={
                Object {
                  "marginLeft": "40%",
                }
              }
            >
              <li
                className=""
                role="presentation"
                style={undefined}
              >
                <a
                  href="#"
                  onClick={[Function]}
                  onKeyDown={[Function]}
                  role="button"
                >
                  <span
                    className="jsx-661576907 description"
                  >
                    <div
                      className="jsx-803102528"
                    >
                      <span
                        color={undefined}
                        style={undefined}
                        variant="raised"
                      >
                        <i
                          className="jsx-803102528 fas fa-trophy"
                          style={
                            Object {
                              "color": "#3ffeca",
                            }
                          }
                        />
                      </span>
                         Contest
                    </div>
                  </span>
                </a>
              </li>
              <li
                className=""
                role="presentation"
                style={undefined}
              >
                <a
                  href="#"
                  onClick={[Function]}
                  onKeyDown={[Function]}
                  role="button"
                >
                  <span
                    className="jsx-661576907 description"
                  >
                    <div
                      className="jsx-803102528"
                    >
                      <span
                        color={undefined}
                        style={undefined}
                        variant="raised"
                      >
                        <i
                          className="jsx-803102528 fas fa-envelope"
                          style={
                            Object {
                              "color": "#3ffeca",
                            }
                          }
                        />
                      </span>
                         Contact
                    </div>
                  </span>
                </a>
              </li>
              <li
                className=""
                role="presentation"
                style={undefined}
              >
                <a
                  href="#"
                  onClick={[Function]}
                  onKeyDown={[Function]}
                  role="button"
                >
                  <span
                    className="jsx-661576907 description"
                  />
                </a>
              </li>
              <li
                className=""
                role="presentation"
                style={undefined}
              >
                <a
                  href="#"
                  onClick={[Function]}
                  onKeyDown={[Function]}
                  role="button"
                >
                  <span
                    className="jsx-661576907 description"
                  />
                </a>
              </li>
            </ul>
          </div>
        </div>
      </nav>
      <nav
        className="navbar navbar-default"
        style={
          Object {
            "background": "#323150",
            "borderColor": "#323150",
            "borderRadius": 0,
            "margin": 0,
          }
        }
      >
        <div
          className="container"
        >
          <div
            className="navbar-header"
          >
            <a
              className="jsx-661576907 navbar-brand"
              href="#home"
              style={undefined}
            />
            <div
              className="jsx-661576907"
              style={
                Object {
                  "float": "right",
                }
              }
            />
            <div
              className="jsx-661576907"
              style={
                Object {
                  "display": "none",
                  "visibility": "hidden",
                }
              }
            >
              <button
                className="navbar-toggle collapsed"
                onClick={[Function]}
                type="button"
              >
                <span
                  className="sr-only"
                >
                  Toggle navigation
                </span>
                <span
                  className="icon-bar"
                />
                <span
                  className="icon-bar"
                />
                <span
                  className="icon-bar"
                />
              </button>
            </div>
          </div>
          <div
            aria-expanded={null}
            className="navbar-collapse collapse"
          >
            <ul
              className="nav navbar-nav"
              role={null}
              style={
                Object {
                  "float": "left",
                }
              }
            >
              <li
                className=""
                role="presentation"
                style={undefined}
              >
                <a
                  href="#"
                  onClick={[Function]}
                  onKeyDown={[Function]}
                  role="button"
                >
                  <span
                    className="jsx-661576907 description"
                  >
                    Home
                  </span>
                </a>
              </li>
              <li
                className=""
                role="presentation"
                style={undefined}
              >
                <a
                  href="#"
                  onClick={[Function]}
                  onKeyDown={[Function]}
                  role="button"
                >
                  <span
                    className="jsx-661576907 description"
                  >
                    Contest 
                  </span>
                </a>
              </li>
              <li
                className=""
                role="presentation"
                style={undefined}
              >
                <a
                  href="#"
                  onClick={[Function]}
                  onKeyDown={[Function]}
                  role="button"
                >
                  <span
                    className="jsx-661576907 description"
                  >
                    FAQ
                  </span>
                </a>
              </li>
              <li
                className=""
                role="presentation"
                style={undefined}
              >
                <a
                  href="#"
                  onClick={[Function]}
                  onKeyDown={[Function]}
                  role="button"
                >
                  <span
                    className="jsx-661576907 description"
                  >
                    Rules
                  </span>
                </a>
              </li>
            </ul>
          </div>
        </div>
      </nav>
    </div>
  </div>
  <div
    className="jsx-2085888330"
  >
    <div
      className="jsx-2085888330"
    >
      <h1
        className="jsx-2085888330"
      >
        Este es el app
      </h1>
    </div>
  </div>
  <p>
    <span>
      Hello 
      <b>
        bitbracket
      </b>
      , you have 1,000 messages
    </span>
  </p>
</div>
`;
